pages.it/common/npx.md

```diff
@@ -0,0 +1,24 @@
+# npx
+
+> Esegue file binari dai pacchetti `npm`.
+> Maggiori informazioni: <https://github.com/npm/npx>.
+
+- Esegue un file binario di uno specifico modulo:
+
+`npx {{nome_modulo}} {{argomenti_comando}}`
```

Comment:
Mandatory fix (no suggestion available): [ellipsis must be used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).
Warning: [no translation template for `{{arg1 arg2 ...}}` found](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md).